staker-rewards/generated-rewards/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "staker-rewards",
+  "description": "Staker KEEP rewards",
```

Comment:
Let's make it clear it's about generating merkle tree for the distribution.

Reply:
Done a330599